src/components/course/routes/ExternalCourseEnrollment.jsx

```diff
@@ -46,15 +70,34 @@ const ExternalCourseEnrollment = () => {
               <h2 className="mb-3">
                 Your registration(s)
               </h2>
-              <p className="small bg-light-500 p-3 rounded-lg">
-                <strong>
-                  This is where you finalize your registration for an edX executive
-                  education course through GetSmarter.
-                </strong>
-                &nbsp; Please ensure that the course details below are correct and confirm using Learner
-                Credit with a &quot;Confirm registration&quot; button.
-                Your Learner Credit funds will be redeemed at this point.
-              </p>
+              {isDuplicateOrder(formSubmissionError) && (
+                <Alert
+                  variant="success"
+                  ref={containerRef}
+                  icon={CheckCircle}
+                  actions={[
+                    <Button href={externalDashboardUrl}>
```

Comment:
Given this hyperlink is going to a new site/domain, we may want to include `_blank` on the hyperlink. You could opt to use `<Button as={Hyperlink} target="_blank" destination={externalDashboardUrl}>Go to dashboard</Button>`.

[Paragon Playground Demo](https://paragon-openedx.netlify.app/playground?code=N4Igxg9gJgpiBcIA8AhArgFwxAdgAgEMBnAXmAAkBPABxgCcAbASxwGsBfPDAugcxgwkAOiAD6AIwYE2IvLCIYWBRbmEgAFlmpF4Ael38MRALY8M9AHSRjugmEhocGAPwQ%2BoplBIANXyIB8AOIQXCFQxOriEDxQSLroWLj%2BIAA0IBjqMMYwRAgA2iAwUN4WbrwgALppAO6eGbnweQCMAEwADG0V7EA)

Reply:
I like this idea!